### PR TITLE
Adding a logo (Google Code-in)

### PR DESCRIPTION
**Pull request by fungktt**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,3 +3,5 @@
 [![Build Status](https://travis-ci.org/JuliaPlots/AbstractPlotting.jl.svg?branch=master)](https://travis-ci.org/JuliaPlots/AbstractPlotting.jl)
 
 An abstract plotting library part of the JuliaPlots/Makie.jl ecosystem.
+
+![logo](https://github.com/fungktt/AbstractPlotting.jl/blob/master/ap-logo.png)
```
Comment (logankilpatrick) [inline README.md]: This file path should not include your GitHub name since this code will live in the master branch of this repo.

Comment (logankilpatrick) [reply README.md]: I would also suggest moving the logo to the top of the README.md file.

Comment (logankilpatrick) [reply README.md]: To clarify, you should essentially link to a location/url in the readme that does not exist right now i.e. "https://github.com/JuliaPlots/AbstractPlotting.jl/tree/master/assets/icons/ap-logo.png"

Comment (logankilpatrick) [reply README.md]: but will exist once this code is merged into master.

File: README.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 # AbstractPlotting
```
Comment (logankilpatrick) [inline README.md]:
```suggestion

```

Comment (logankilpatrick) [reply README.md]: I would remove this since the logo does the job now.

**Comment by asinghvi17** [comment]
Thanks @fungktt!  Logo looks really cool - just two things I wanted to bring up.

- Could you rename this to `logo.png`?  It plays better with Documenter.
- I'm not sure about the multiple colours - would it make sense to  have `Abstract` in the first Julia logo colour, `Plotting` in the second, and `.jl` in the third?

**Comment by asinghvi17** [comment]
Awesome!  One final thing - could you remove the blue underline?  Will merge after that.

**Comment by asinghvi17** [comment]
@fungktt could you post a brief description / the source code of how you made the logo?  Thanks!

**Pull request merged**